tests/test_salesforce_discovery_rest.py

```diff
@@ -139,7 +139,7 @@ def discovery_test(self):
                     'ApiEvent', 'WorkOrder', 'ContactCleanInfo', 'ResourceAbsence', 'ReturnOrder',
                     'LegalEntity', 'PaymentMethod', 'EventLogFile', 'ServiceAppointment',
                     'DandBCompany', 'AccountCleanInfo', 'Organization', 'Document', 'Account',
-                    'Address', 'FulfillmentOrder'
+                    'Address', 'FulfillmentOrder', 'Asset'
```

Comment:
Asset stream contains the `Address` field which has `inclusion=unsupported` in v61. So, skipping it for this check.